tests/integration/kubernetes/k8s-guest-pull-image.bats

```diff
@@ -9,7 +9,9 @@ load "${BATS_TEST_DIRNAME}/lib.sh"
 load "${BATS_TEST_DIRNAME}/confidential_common.sh"
 
 setup() {
-    confidential_setup && skip "Due to issues related to pull-image integration skip tests for ${KATA_HYPERVISOR}."
+    if [ is_confidential_runtime_class ]; then
+        skip "Due to issues related to pull-image integration skip tests for ${KATA_HYPERVISOR}."
```

Comment:
I don't think this is correct - we need the 
```
if [ ! is_confidential_runtime_class ]; then
		skip "Test not supported for ${KATA_HYPERVISOR}."
	fi
```
here to stop the e.g. clh tests from running?

Reply:
Ok, it was fixed in the later commit, so just this commit is wrong

Reply:
Yeah, the first commit is just changing the name, thus it'd be wrong (as the current logic is wrong).
The logic itself is fixed in the 3rd commit.